agent.py

```diff
@@ -149,16 +155,34 @@ def explore(self, dt, fov):
             if (0 < d[0] < self.perception_front_dims[0]) and \
                     d[0] < d_nearest_target:
                 if 0 < d[1] < self.perception_front_dims[1]:
-                    evade_direction = 1
+                    if target[1] > \
+                            (635 - (2 * self.perception_front_dims[1])) and \
```

Comment:
pls fix all magic values, you fuckers

Reply:
![afbeelding](https://github.com/Finnyy012/TUPOLEVGAMING/assets/94832237/527324e4-72eb-49ce-9f6d-8c118f2c49ae)